openedx/stanford/djangoapps/auth_lagunita/forms.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django import forms
+
+from .models import Info
+
+
+class InfoForm(forms.ModelForm):
+    """
+    The fields on this form are derived from the Info model
+    """
+    def __init__(self, *args, **kwargs):
+        super(InfoForm, self).__init__(*args, **kwargs)
+
+    class Meta(object):
+        fields = (
+            'requested_marketing_optin',
+        )
+        model = Info
+
+    requested_marketing_optin = forms.BooleanField(
+        label=(
+            'I want to receive email about future learning opportunities '
+            'and courses from Stanford Online '
+            'and I am at least 18 years old.'
```

Comment:
@jen-nifer What text do we want to go with?

Reply:
Please use "I want to receive email about future learning opportunities from Stanford Online and I am at least 18 years old."

Reply:
Text updated.